imports/ui/components/pwd-strength.js

```diff
@@ -1,42 +0,0 @@
-/* Password strength validator */
```

Comment:
Why did you remove it from here and added in the other archive? I think that if we can separate this logic into another file it is better organized